tests/vmcheck/multitest.py

```diff
@@ -13,11 +13,32 @@ def main():
     for host in sys.argv[1:]:
         hosts.append(Host(host))
 
-    for test in glob.iglob(os.path.join(sys.path[0], "test-*.sh")):
+    requested_tests_spec = os.environ.get('TESTS')
+    if requested_tests_spec is not None:
+        requested_tests = requested_tests_spec.split()
+    else:
+        requested_tests = None
+
+    tests = glob.iglob(os.path.join(sys.path[0], "test-*.sh"))
+    matched_tests = []
+    unmatched_tests = []
+    for test in tests:
+        testbn = os.path.basename(test)
```

Comment:
Can we use `_strip_test()` for this? With `test.sh`, the actual syntax for `TESTS` excludes the leading "test-" and trailing ".sh" (just like the `test.sh` in projectatomic/atomic). We should probably accept the same syntax here too.